templates/3/modules/ldap.erb

```diff
@@ -123,34 +121,23 @@ ldap {
   }
 
   options {
-<% if @options_chase_referrals -%>
-    chase_referrals = yes
-<% else -%>
-    chase_referrals = no
-<% end -%>
-<% if @options_rebind -%>
-    rebind = yes
-<% else -%>
-    rebind = no
-<% end -%>
-    timeout = <%= @ldap_timeout %>
-    timelimit = <%= @ldap_timelimit %>
+    dereference = <%= @options_dereference %>
+    chase_referrals = <%= bool2str[@options_chase_referrals] %>
+    rebind = <%= bool2str[@options_rebind] %>
+    res_timeout = <%= @ldap_timeout %>
+    srv_timelimit = <%= @ldap_timelimit %>
     net_timeout = <%= @options_net_timeout %>
     idle = <%= @options_idle %>
     probes = <%= @options_probes %>
     interval = <%= @options_interval %>
-<% if not @ldap_debug.eql?('nil') then -%>
-    ldap_debug = <%= @ldap_debug %>
+<% if @ldap_debug -%>
+    ldap_debug = "<%= @ldap_debug %>"
```

Comment:
Why is this in double quotes when all the other delimited strings are in single quotes?

Reply:
To be addressed under [SIMP-6177](https://simp-project.atlassian.net/browse/SIMP-6177)